src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -132,6 +164,12 @@ public void resetModulesToAbsolute(){
             mod.resetToAbsolute();
         }
     }
+    public ChassisSpeeds getRobotRelativeSpeeds(){
+        return ChassisSpeeds.fromRobotRelativeSpeeds(getRobotRelativeSpeeds(), getGyroYaw());
```

Comment:
Is this recursive? If so, I'm guessing we need to rethink where we're getting ChassisSpeeds for this call.